messaging-activemq/injection/src/main/java/org/wildfly/extension/messaging/activemq/_private/MessagingLogger.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.messaging.activemq._private;
+
+import jakarta.jms.IllegalStateRuntimeException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Date: 10.06.2011
+ *
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ */
+@MessageLogger(projectCode = "WFLYMSGAMQINJ", length = 4)
+public interface MessagingLogger extends BasicLogger {
+    /**
+     * The logger with the category of the package.
+     */
+    MessagingLogger ROOT_LOGGER = Logger.getMessageLogger(MessagingLogger.class, "org.wildfly.extension.messaging-activemq");
+
+    /**
+     * Create an exception when calling a method not allowed on injected JMSContext.
+     *
+     * @return an {@link IllegalStateRuntimeException} for the error.
+     */
+    @Message(id = 1, value = "(WFLYMSGAMQ0068) It is not permitted to call this method on injected JMSContext (see Jakarta Messaging 2.0 spec, §12.4.5).")
```

Comment:
Adding "(WFLYMSGAMQ0068)" isn't a bad idea, but do we just want to do a clean cut? I've got no strong opinion either way, but if we add it, it will likely be there forever.

Reply:
I'm the one who suggesting adding this in the zulip thread, but now that I've read what this message actually says I'm fine with a complete cut. I think it's very unlikely that this id is important as a search term, since the message itself (and the associated stack trace) is very precise and points to the spec section where readers can learn more. I don't see what any search result would add to that.

I don't think it does any harm either. The WFLYMSGAMQ0068 id is burned so it's never going to be reused.

My vote is @soul2zimate and @ehsavoie decide what they want and that's fine. :)

Reply:
This makes sense to me. I don't think it hurts anything having it really, just wondered if we wanted to cut it out now or live with it forever is all :)

Reply:
@ehsavoie I'd like to remove `(WFLYMSGAMQ0068)` prefix in the message to have a clean cut unless you have an opposite preference

Reply:
@soul2zimate I agree.

Reply:
Thanks @ehsavoie @bstansberry @jamezp . I have removed `(WFLYMSGAMQ0068)` prefix in the message